<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="DESKTOP-A956FE6" name="org.test3.AdactinDataProvider" tests="1" failures="0" timestamp="16 Mar 2023 07:07:05 GMT" time="0.028" errors="1">
  <testcase name="login" time="0.028" classname="org.test3.AdactinDataProvider">
    <error type="org.testng.internal.reflect.MethodMatcherException" message="
Data provider mismatch
Method: login([Parameter{index=0, type=java.lang.String, declaredAnnotations=[]}, Parameter{index=1, type=java.lang.String, declaredAnnotations=[]}])
Arguments: [(java.lang.String) Abiraj97,(java.lang.String) Qwerty@123,(java.lang.String) Sydney,(java.lang.String) Hotel Sunshine,(java.lang.String) Double,(java.lang.String) 5,(java.lang.String) 2,(java.lang.String) 2,(java.lang.String) Abi,(java.lang.String) Raj,(java.lang.String) Chennai,(java.lang.String) 4869509087687612,(java.lang.String) VISA,(java.lang.String) April,(java.lang.String) 2017,(java.lang.String) 543]">
      <![CDATA[org.testng.internal.reflect.MethodMatcherException: 
Data provider mismatch
Method: login([Parameter{index=0, type=java.lang.String, declaredAnnotations=[]}, Parameter{index=1, type=java.lang.String, declaredAnnotations=[]}])
Arguments: [(java.lang.String) Abiraj97,(java.lang.String) Qwerty@123,(java.lang.String) Sydney,(java.lang.String) Hotel Sunshine,(java.lang.String) Double,(java.lang.String) 5,(java.lang.String) 2,(java.lang.String) 2,(java.lang.String) Abi,(java.lang.String) Raj,(java.lang.String) Chennai,(java.lang.String) 4869509087687612,(java.lang.String) VISA,(java.lang.String) April,(java.lang.String) 2017,(java.lang.String) 543]
at org.testng.internal.reflect.DataProviderMethodMatcher.getConformingArguments(DataProviderMethodMatcher.java:45)
at org.testng.internal.Parameters.injectParameters(Parameters.java:796)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:983)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- login -->
</testsuite> <!-- org.test3.AdactinDataProvider -->
